diff --git a/client/meson.build b/client/meson.build
--- a/client/meson.build
+++ b/client/meson.build
@@ -5,7 +5,7 @@
 cppc = meson.get_compiler('cpp')

 use_fortran = false
-if get_option('with_fortran') or get_option('with_xtb') or get_option(
+if get_option('with_fortran') or get_option(
     'with_cuh2',
 )
     use_fortran = true
@@ -306,9 +306,8 @@
 endif

 if get_option('with_xtb')
-    xtb_proj = subproject('xtb')
-    xtb_dep_static = xtb_proj.get_variable('xtb_dep_static')
-    _deps += [xtb_dep_static]
+    xtb_dep = dependency('xtb', required: true)
+    _deps += [xtb_dep]
     subdir('potentials/XTBPot')
     potentials += xtb_eon
     _args += ['-DWITH_XTB']
@@ -409,7 +408,10 @@
     lib_torch_list = ['c10', 'torch', 'torch_cpu', 'torch_global_deps']
     tdeps = []
     foreach lib_name : lib_torch_list
-        tdeps += cppc.find_library(lib_name, dirs: [LIB_TORCH_LIB_PATH])
+        _lib = cppc.find_library(lib_name, dirs: [LIB_TORCH_LIB_PATH], required: false)
+        if _lib.found()
+            tdeps += _lib
+        endif
     endforeach

     torch_inc_args = []
