diff --git a/client/meson.build b/client/meson.build
--- a/client/meson.build
+++ b/client/meson.build
@@ -4,8 +4,8 @@ cc = meson.get_compiler('c')
 cppc = meson.get_compiler('cpp')

 use_fortran = false
-if get_option('with_fortran') or get_option('with_xtb') or get_option(
-    'with_cuh2',
+if get_option('with_fortran') or get_option(
+    'with_cuh2',
 )
     use_fortran = true
     add_languages('fortran', required: true)
